III.tdd-katas/calculator-tdd/add.test.js

```diff
@@ -0,0 +1,40 @@
+const add = require("./add");
+describe("Step 2: handle an unknown amount of numbers", () => {
+    test("should return 0 for empty string passed to add", () => {
+        expect(add("")).toBe(0);
+    });
+    test('should return 1 for "1" string passed to add', () => {
+        expect(add("1")).toBe(1);
+    });
+    test('should return 3 for "1,2" string passed to add', () => {
+        expect(add("1,2")).toBe(3);
+    });
+});
+describe("Step 2: handle an unknown amount of numbers", () => {
+    test("should return 0 for empty string passed to add", () => {
+        expect(add("1,2,4fi5faswwe6,lgsf8")).toBe(26);
+    });
+});
+describe("Step 3: handle new lines between numbers", () => {
+    test("should return 26 for the string passed to add", () => {
+        expect(add("1,2,4fi5faswwe6,lgsf8")).toBe(26);
+    });
+});
+describe("Step 4: support different delimiters", () => {
+    test("should return 9.3 for the string passed to add", () => {
+        expect(add("//3[$5.3delimiter]\n[numbers..&.*1]")).toBeCloseTo(9.3);
```

Comment:
I don't think this test is correct - if the first line contains `//3[$5.3delimiter]` then the calculator should split the numbers in the second line based on *that whole string*, `3[$5.3delimiter]`. Look again at the example given: 

> For example `"//;\n1;2"` should return 3 where the default delimiter is `';'`.